src/foam/nanos/controller/ApplicationController.js

```diff
@@ -224,6 +225,13 @@ foam.CLASS({
       name: 'displayWidth',
       value: foam.u2.layout.DisplayWidth.XL
     },
+    {
+      class: 'Boolean',
+      name: 'isDesktop',
+      expression: function(displayWidth) {
+        return displayWidth.minWidth > 320;
```

Comment:
Should be using the DisplayWidth enum, we should also do this in the view that needs this change, a generalized description of isDesktop is not going to work for many components